### PR TITLE
RAD-Moon3: Phy support (v913)

### PR DESCRIPTION
**Pull request by FairosAws**
#140

### CODE REVIEW

File: src/methods.cpp [hunk 1]
```diff
@@ -3299,9 +3299,9 @@ PyObject* meth_get_phy_firmware_version(PyObject* self, PyObject* args)
 {
 	PyObject* obj = NULL;
 	char phy_indx = 0;
-	unsigned int phyVersion = NULL;
-	unsigned int functionError = 1;
-	if (!PyArg_ParseTuple(args, arg_parse("Oi|ii:", __FUNCTION__), &obj, &phy_indx, &phyVersion, &functionError))
+	int phy_version = NULL;
+	int function_error = 1;
```
Comment (drebbe-intrepid) [inline src/methods.cpp:3303]: `phy_version` and `function_error` shouldn't be passed into `PyArg_ParseTuple`. Python doesn't send these as references/pointers.

File: src/methods.cpp [hunk 2]
```diff
@@ -3321,15 +3321,15 @@ PyObject* meth_get_phy_firmware_version(PyObject* self, PyObject* args)
 			return set_ics_exception(exception_runtime_error(), dll_get_error(buffer));
 		}
 
-		ice::Function<int __stdcall (ICS_HANDLE, char, unsigned int*, unsigned int*)> icsneoGetPhyFwVersion(lib, "icsneoGetPhyFwVersion");
+		ice::Function<int __stdcall (ICS_HANDLE, char, int*, int*)> icsneoGetPhyFwVersion(lib, "icsneoGetPhyFwVersion");
 
         Py_BEGIN_ALLOW_THREADS
-            if (!icsneoGetPhyFwVersion(handle, phy_indx, &phyVersion, &functionError)) {
+            if (!icsneoGetPhyFwVersion(handle, phy_indx, &phy_version, &function_error)) {
```
Comment (drebbe-intrepid) [inline src/methods.cpp:3327]: we should be checking against `function_error` and raising an exception if its not valid.

**Comment by drebbe-intrepid** [comment]
#143

**Comment by drebbe-intrepid** [comment]
Slated for 913 release

**Comment by drebbe-intrepid** [comment]
This won't compile until we update the header to v913.

**Comment by drebbe-intrepid** [comment]
@FairosAws this is ready to be tested

**Comment by FairosAws** [comment]
> @FairosAws this is ready to be tested

All good, ready to merge

**Comment by FairosAws** [comment]
> @FairosAws this is ready to be tested

All good, ready to merge

**Pull request merged**